CHANGELOG.md

```diff
@@ -6,6 +6,19 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+### Changed
+- **CUMULUS-3077**
+  - Updated `lambdas/data-migration2` granule and files migration to have a `removeExcessFiles` function like in write-granules that will remove file records no longer associated with a granule being migrated
+
```

Comment:
Only need to keep entries from https://github.com/nasa/cumulus/pull/3134

Reply:
@jennyhliu I'll remove mine for 3045 in the "Changed" section but his entry for 3077 is "as is" state. That's how the changelog looked when he submitted it. His is up under "Unreleased" and the "Changed" section further down.